### DISCUSSION TITLE
More Pictographic Options

### DISCUSSION THREAD

**Discussion by PhoneDroid** [pulsar-edit/.github#120] [Pulsar Features]
<br>

I'd like to suggest using more pictography for user 
actions or at least make them available as an option.

<br>
<br>

## Example

The close options not only take up a lot
of space, but also require you to scan 
over all of them to find the one you want,
the same problem exists in VSCode btw.

![Close](https://user-images.githubusercontent.com/73050054/215343601-20a15ee4-b6fe-490b-9781-8dcaec1087e9.png)

<br>

### Possible Options

*Can be combined*

- Use icons for each action

- Have a left aligned descriptor Close

- Preview each option when hovering over them
  ( Indicate which tabs would close, by for example 
  coloring the background of the tab header )

- Make it adjustable which options are displayed

- Don't display options that have no effect 

<br>

### Simple Demo

```
+-------+----+----+----+----+----+----+
| Close | ◀  | 🔒 | 📖 | 📚 | 📑 |  ▶ |
+-------+----+----+----+----+----+----+
```

*Icons just for demonstration / unrelated*

<br>

**Comment by Daeraxa** [comment]
I opened [this topic](https://github.com/orgs/pulsar-edit/discussions/86) before about an Octicons replacement (and/or update) which would lend this this kind of idea quite nicely in terms of icon variety.

There are definitely aspects of Pulsar that feel very dated and this is one of them, we have quite a few areas with excessive menu items that all look very similar and are hard to distinguish at a glance.

**Reply by kaosine** [reply]
Agreed, this is probably one way we could modernize and update the interface, or at least give people a good option to change between the two. Say on the welcome screen after it were implemented and potentially in a `view ->layout` or `view->interface style` type thing. It would be a good way to give people options and also modernize and make things easier for others.

(with a hopefully better labeled name than what I put just then...I know there's a better name for it, but it definitely should be in a view menu w/it also in the settings as well)
